packages/emails/email-manager.ts

```diff
@@ -43,19 +48,37 @@ export const sendScheduledEmails = async (calEvent: CalendarEvent) => {
   emailsToSend.push(
     new Promise((resolve, reject) => {
       try {
-        const scheduledEmail = new OrganizerScheduledEmail(calEvent);
+        const scheduledEmail = new OrganizerScheduledEmail(clonedEvent);
         resolve(scheduledEmail.sendEmail());
       } catch (e) {
         reject(console.error("OrganizerScheduledEmail.sendEmail failed", e));
       }
     })
   );
 
+  if (clonedEvent.team) {
+    for (const teamMember of clonedEvent.team.members) {
+      emailsToSend.push(
+        new Promise((resolve, reject) => {
+          try {
+            const scheduledEmail = new OrganizerScheduledEmail(clonedEvent, undefined, teamMember);
+            resolve(scheduledEmail.sendEmail());
+          } catch (e) {
+            reject(console.error("OrganizerScheduledEmail.sendEmail failed", e));
+          }
+        })
+      );
+    }
```

Comment:
```suggestion
const sendMail = (prepare ) => {
  return new Promise((resolve, reject) => {
    try {
      const email = prepare();
      resolve(email.sendEmail());
    } catch (e) {
      reject(console.error("sendEmail failed", e));
    }
  });
};
      emailsToSend.push(sendMail(()=>new OrganizerScheduledEmail(clonedEvent, undefined, teamMember)))
```

Reply:
I really liked this idea so I created a separate PR #7270